ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -123,14 +123,14 @@ protected override async Task<bool> ValidateAndSave()
 
         private void UpdateFlags()
         {
-            if (ClinicalDetails.DidNotStartTreatment)
+            if (ClinicalDetails.DidNotStartTreatment ?? false)
             {
                 ClinicalDetails.TreatmentStartDate = null;
                 FormattedTreatmentDate = ClinicalDetails.TreatmentStartDate.ConvertToFormattedDate();
                 ModelState.Remove("ClinicalDetails.TreatmentStartDate");
             }
 
-            if (!ClinicalDetails.IsPostMortem)
+            if (!ClinicalDetails.IsPostMortem ?? false)
```

Comment:
Negation with a null coalesce makes this a little tricky to scan.

I think the following would be more readable - but definitely a stylistic choice and am happy to be overruled by your judgement on this:

`ClinicalDetails.DidNotStartTreament == true` and `ClinicalDetails.DidNotStartTreatment != true`

Reply:
we had a mix of == true and ?? false for nullable booleans and then decided to default to ?? false  @jasiekmiko mentioned it in one of the comments, but I am happy to default back to == true if we agree on it. But otherwise, I don't think it's too bad to understand

Reply:
Yeah, i did previously steer from `== true` to `??`, but Conor makes a really good point of it being quite confusing in conjuction with `!` - especially that the order is not super clear!
Happy to go with that suggestion